entrypoint.sh

```
@@ -457,4 +457,183 @@ echo "   Total time: $((UPLOAD_DURATION + CONFIRM_DURATION)) seconds"
+FLOW_RUN_COUNT=$(echo "$JSON_RESPONSE" | jq -r '.flow_run_count')
+
+echo "✅ Triggered $FLOW_RUN_COUNT flow(s)"
+echo "   Batch ID: $BATCH_ID"
+echo ""
+echo "⏳ Waiting for results..."
+echo ""
+
+# Polling configuration
+POLL_INTERVAL=15
+PRINTED_IDS_FILE=$(mktemp)
+trap "rm -f $PRINTED_IDS_FILE" EXIT
```

Shellcheck (0.11.0)

[warning] Use single quotes, otherwise this expands now rather than when signalled.

(SC2064)